modules/ROOT/pages/migration-connectors-http.adoc

```diff
@@ -614,8 +616,11 @@ in the HTTP response body using DataWeave, where you can easily customize header
 It's important to notice that while we are taking advantage of the `http:body` feature,
 the same result could be achieved using a final transform component.
 
+Note that the content type header is autogenerated correctly. For details, see
```

Comment:
I'd say: `Note that the boundary is autogenerated, which results in a correct content-type header.` Because the header is not the one autogenerated.